src/components/ShareTrajectoryButton/index.tsx

```diff
@@ -30,7 +30,7 @@ const ShareTrajectoryButton = ({
     const tooltipOffset = isDisabled ? [0, -30] : [0, -18];
 
     return (
-        <div className={styles.container}>
+        <div className={styles.container} onClick={handleShare}>
```

Comment:
it's a little bit of a weird thing since this is wrapping another button. I think the Dom would have a problem with a button within a button? but I can try

Reply:
yeah, ```validateDOMNesting(...): <button> cannot appear as a descendant of <button>.```

Reply:
Is there a reason why the `onClick` got moved to the div?

Reply:
Yeah, if you try it out on staging the click area is tiny. I tried other fixes but this was the only thing that got the target to be the whole button

Reply:
Okay... maybe try adding a role attribute to signal that the div can be treated like a button? https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role

I opened up the Simularium website and realized the entire site doesn't seem to work well with tab navigation, so there may be bigger fish to fry...

Reply:
your questions made me dive deeper into what was causing the button to be unclickable and I realized it was the tooltip placement. when the tooltip opens it actually covers the target because it had been aligned over the top. So I reverted this change and fixed that on a [separate branch.](https://github.com/simularium/simularium-website/pull/469)